common/onepanel/base/vars.yaml

```diff
@@ -78,4 +78,17 @@ application:
     # CLI flag: --enable-https
     insecure:
       required: true
-      default: false
\ No newline at end of file
+      default: false
+  # Node pool key/values are used for nodeSelector and autoscaling.
+  nodePool:
+    label: <node-pool-label>
+    # Add more by following the format
+    # - name: <name>
+    #   value: <value>
+    #   default: <true|false>
+    options:
```

Comment:
Can't we just do a multiline string here so we don't have to do [this](https://github.com/onepanelio/cli/pull/36/files#diff-ce0151979a0da3cf1233dd0fd63a5aa8R449-R475) in the CLI?

Reply:
Ok, I just checked the generated output in `params.yaml` and you're actually generating a YAML instead of a multi-line string. For some reason, I thought it was a multiline string based on your [comment here](https://github.com/onepanelio/cli/pull/36#issue-405383238)